docs/source/configuration/environmentvariables.md

```diff
@@ -9,193 +9,190 @@ myst:
 
-ex:
-If we start the application with an env variable
 
-```bash
+## Access environment variables
+
+All environment variables defined at runtime with the "RAZZLE_" prefix are available in your browser under the `window.env` global object.
+For example, start the application with an environment variable as shown.
+
+```shell
 RAZZLE_MY_VARIABLE=some_value build/server.js
 ```
 
-In the frontend we can access this variable with:
+In the frontend, you can access this variable in your code with the following.
 
-```bash
+```shell
 window.env.RAZZLE_MY_VARIABLE
 ```
 
-## Runtime environment variables
 
-All the environment variables that are configurable work at runtime, not only at build time. This works since Volto 13 onwards.
-
-````{note}
-Before Volto 13, you'd do:
+## Runtime environment variables
 
-```bash
-RAZZLE_API_PATH=https://plone.org yarn build && yarn start:prod
+```{versionadded} 13
 ```
 
-From Volto 13 onwards, you can now do:
+All configurable environment variables work at runtime, not only at build time.
+You could, for example, build your Volto application, then start it in production with the `RAZZLE_API_PATH` environment variable.
 
-```bash
+```shell
 yarn build && RAZZLE_API_PATH=https://plone.org yarn start:prod
 ```
-````
-
-This brings you a lot of power since you don't have to rebuild on every config change. You can also generate builds on your CI, then deploy them anywhere.
-
-
-## Environment variables reference
-
-### RAZZLE_LEGACY_TRAVERSE
-
-From Volto 14 onwards, Seamless mode is the recommended way of setting up your deployments. However, it forces you to upgrade several packages in the backend (`plone.restapi` and `plone.rest`) and adjust your web server configuration accordingly.
-
-In case you can't afford or change your deployment, you can still upgrade Volto by using the `RAZZLE_LEGACY_TRAVERSE` flag.
-
-```bash
-RAZZLE_LEGACY_TRAVERSE=true yarn start:prod
-```
-
-### VOLTO_ROBOTSTXT
-
-You can override the robots.txt file with an environment variable called
-`VOLTO_ROBOTSTXT`. This is useful when using the same build on multiple
-websites (for example a test website) to forbid robots from crawling it.
-
-```
-$ VOLTO_ROBOTSTXT="User-agent: *
-Disallow: /" yarn start
-```
 
-```{note}
-If you want to use the `VOLTO_ROBOTSTXT` environment variable, make sure to
-delete the file `public/robots.txt` from your project.
-```
+This brings you a lot of power since you don't have to rebuild on every configuration change.
+You can also generate builds on your continuous integration, then deploy them anywhere.
 
-### DEBUG
 
-It will enable the log several logging points scattered through the Volto code. It uses the `volto:` namespace.
+## Environment variable reference
 
-```bash
-DEBUG=volto:i18n yarn start
-```
+````{glossary}
+:sorted:
+`RAZZLE_LEGACY_TRAVERSE`
+    {doc}`../deploying/seamless-mode` is the recommended way of setting up your deployments since Volto 14.
```

Comment:
```suggestion
    If `true`, Volto will construct API URLs without the `/++api++` prefix.

    This is not needed if you are using {doc}`../deploying/seamless-mode`, which is the recommended way to set up your deployments since Volto 14.
```